### PR TITLE
[i18n-dev] update crontab

### PR DESCRIPTION
**Pull request by elf-code**
I've been iterating on fixing the crontab on the i18n-dev server, this is the crontab that is currently on the server, so checking in the changes.

You can edit the crontab directly on the server with `crontab -e`

I think this will overwrite the local changes when chef runs.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: experimental/i18n-dev.crontab [hunk 1]
```diff
@@ -15,21 +15,21 @@
 #
 # Where can I ask a question about this config?
 # Reach out to the #i18n slack channel.
-
+PATH=/home/ubuntu/.rbenv/shims:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```
Comment (nicklathe) [inline experimental/i18n-dev.crontab]: Yeah, this is what was previously being done. It works, my only thing is from a discovery standpoint, it's a bit tricky that this PATH will override the one being set in the `~/.bashrc` file, which is what I was trying to get cron to pick up. But this works, so I say we just do it this way and be done with it.

File: experimental/i18n-dev.crontab [hunk 2]
```diff
@@ -15,21 +15,21 @@
...
 HOME=/home/ubuntu
 
 # Run a CI Build to update the server from staging. We do this primarily to
 # make sure the server gets seeded with latest level content for the sync in.
 # Note that the CI_ONLY_BUILD environment variable is necessary; the server is
 # not set up to actually serve a version of the website, and it will fail to
 # build correctly if not thus restricted.
-0 */4 * * * BASH_ENV=~/.bashrc bash -lc "cd /home/ubuntu/code-dot-org && CI_ONLY_BUILD=true bundle exec ./bin/cronjob ./aws/ci_build >>/tmp/cronlog 2>&1"
+0 */4 * * * bash -l -c "cd /home/ubuntu/code-dot-org && CI_ONLY_BUILD=true bundle exec ./bin/cronjob ./aws/ci_build >>/tmp/cronlog 2>&1"
```
Comment (nicklathe) [inline experimental/i18n-dev.crontab:26]: I'm actually not sure we need `bash -l -c` anymore if we are setting the PATH variable directly in this crontab file?

File: experimental/i18n-dev.crontab (hunk 1 above)
Comment (elf-code) [reply experimental/i18n-dev.crontab]: One thing I noticed was that `~/.bashrc` *didn't* include a $PATH update .. so while I understand that from a discovery point of view, `~/.bashrc` is where $PATH updates should be, it wasn't.

File: experimental/i18n-dev.crontab (hunk 2 above)
Comment (elf-code) [reply experimental/i18n-dev.crontab:26]: I agree.. but now that it's working, I don't want to mess it up. 😂

**Review by nicklathe — APPROVED** [review]
LGTM

File: experimental/i18n-dev.crontab (hunk 1 above)
Comment (nicklathe) [reply experimental/i18n-dev.crontab]: Oh, one more thing. When I `echo $PATH`, nvm is present, but I don't see it being added here in the hardcoded PATH. I think we need to add: `/home/ubuntu/.nvm/versions/node/v18.16.0/bin` to the PATH variable here.

**Review by mgc1194 — APPROVED** [review]
Hopefully these changes finally fix the issues.

**Pull request merged**